CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Changes since v2.17
 
 ### Changes
 - The "Attachments (zip)" button in the UI now only downloads the current application attachments. Event attachments and previous versions of application attachments are left out. The full zip is still available via the API. (#2453)
+- Changes to theming: (#2588)
+  - Theme variables are now documented in [resources/config-defaults.edn](resources/config-defaults.edn).
+  - The theme variables `:danger-color` and `:phase-background-active` have been removed.
```

Comment:
Could add here that they weren't used or useful?

Reply:
I'll do that